ui/components/ui/box/box.stories.js

```diff
@@ -324,31 +425,28 @@ export const BorderColor = () => {
         padding={3}
         backgroundColor={COLORS.BACKGROUND_DEFAULT}
         borderColor={COLORS.BORDER_MUTED}
+        borderWidth={2}
+        marginBottom={1}
       >
         <Typography color={COLORS.TEXT_DEFAULT}>COLORS.BORDER_MUTED</Typography>
       </Box>
       <Box
         padding={3}
         borderColor={COLORS.PRIMARY_DEFAULT}
+        borderWidth={2}
+        marginBottom={1}
```

Comment:
Increasing borderWidth here so it's more apparent and easier to see

## Before

![Uploading Screen Shot 2022-08-11 at 4.25.54 PM.png…]()


## After
<img width="1440" alt="Screen Shot 2022-08-11 at 4 25 24 PM" src="https://user-images.githubusercontent.com/8112138/184258709-bd297bbc-7738-4763-90c2-f567de3070fa.png">